internal/db/schema/migrations/oss/postgres/48/01_static_kv_creds.up.sql

```diff
@@ -0,0 +1,81 @@
+begin;
+
+  -- drop constraint so we can add kv
+  alter table credential_type_enm
+    drop constraint only_predefined_credential_types_allowed;
+
+  -- Add new constraint that only allows known types
+  -- This replaces the constraint defined in 39/01_static_ssh_private_key_creds.up.sql
+  alter table credential_type_enm
+    add constraint only_predefined_credential_types_allowed
+      check (
+        name in (
+          'unspecified',
+          'username_password',
+          'ssh_private_key',
+          'kv'
```

Comment:
Why does `kv` need to be added to the credential types?

Reply:
I see, credential types are only used as a foreign key constraint for the credential library